src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import mate.academy.spring.validator.Email;
+import mate.academy.spring.validator.ValidPassword;
+
+@ValidPassword
 public class UserRequestDto {
+    @Email
     private String email;
     private String password;
+    private String repeatPassword;
```

Comment:
min size would be good to have too

Reply:
I use regex to validate a password and min length is already set (8) there.

Reply:
![Screenshot 2023-03-10 225147](https://user-images.githubusercontent.com/42543757/224426132-4dba29e2-7dcb-43d9-a6ad-f0416724da14.jpg)